option/internaloption/internaloption.go

```diff
@@ -163,6 +166,10 @@ func (w withDefaultUniverseDomain) Apply(o *internal.DialSettings) {
 
 // EnableJwtWithScope returns a ClientOption that specifies if scope can be used
 // with self-signed JWT.
+//
+// EnableJwtWithScope is ignored when option.WithUniverseDomain is set
+// to a non-GDU universe domain. For non-GDU universe domains, token exchange is
```

Comment:
What is a GDU, would a user know this term?

Reply:
Good call, I'll expand it.

Reply:
Actually, this is internaloption, is it ok here? Or do you think I should spell it out for maintainers as well.

Reply:
I think it is still worth while as I am not sure what it stand for

Reply:
"Google Default Universe" fwiw

Reply:
Fixed, thanks!